src/Maybe/getPath.js

```diff
@@ -0,0 +1,55 @@
+/** @license ISC License (c) copyright 2019 original and current authors */
+/** @author Ian Hofmann-Hicks (evil) */
+
+const { Nothing, Just } = require('../core/Maybe')
+
+const curry = require('../core/curry')
+const isArray = require('../core/isArray')
+const isDefined = require('../core/isDefined')
+const isEmpty = require('../core/isEmpty')
+const isInteger = require('../core/isInteger')
+const isNil = require('../core/isNil')
+const isString = require('../core/isString')
+
+function fn(name) {
+  function getPath(keys, target) {
+    if(!isArray(keys)) {
+      throw new TypeError(`${name}: Array of Non-empty Strings or Integers required for first argument`)
```

Comment:
```suggestion
      throw new TypeError(`${name}: First argument must be an Array of non-empty Strings or Integers`)
```

We use a mix of voices across the project in these errors. I prefer moving away from passive forms like "is required"